modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -627,21 +648,20 @@ describe('TSS Ecdsa Utils:', async function () {
         }),
         reqId,
       }).should.be.rejectedWith('Invalid user key - missing backupNShare');
-
-      sandbox.verifyAndRestore();
     });
 
-    async function setupSignTxRequestNocks(isTxRequest = true) {
+    async function setupSignTxRequestNocks(isTxRequest = true, userSignShare: ECDSA.SignShareRT, aShare: ECDSA.AShare, dShare: ECDSA.DShare) {
+      sinon.stub(ECDSAMethods, 'createUserSignShare').resolves(userSignShare);
```

Comment:
Cheating a bit here, but this prevents us from having to do an absolute boatload of stubbing to every bit of randomness required in the range proof stuff. I think it's fine, as ostensibly this is testing if the flow of `signRequestBase()` works, but open to differing opinions here